test/focusable.test.js

```diff
@@ -293,6 +294,68 @@ describe('focusable', () => {
           expectedFocusableIds
         );
       });
+
+      describe('displayed check', () => {
```

Comment:
Maybe this was copy/paste, but could you find/replace "Tabbable" with "Focusable" in this whole block? Just confusing to reach variable names with "Tabbable", yet we're testing `focusable()` here. 😄

Reply:
good catch!